1.0/docs/start/psk/set-up.md

```diff
@@ -0,0 +1,207 @@
+---
+layout: default
+type: start
+shortname: Start
+title: Set up
+subtitle: Using Polymer Starter Kit
+---
+
+{% include toc.html %}
+
+The Polymer Starter Kit is a complete starting point for a Polymer 1.0 
+application. It includes:
+
+*   A [Material Design][md] layout
+*   Routing with Page.js
+*   Web component unit testing
+*   An end-to-end build and local development toolchain
+
+Follow the instructions below to install, build, and run the 
+Polymer Starter Kit (PSK) locally in less than 15 minutes.
+
+## Install the Polymer Starter Kit and dependencies
+
+1.  Install [Node.js](https://nodejs.org/) (`node`) version 0.12 or above. 
+    Node.js includes Node Package Manager (`npm`) by default. The PSK 
+    uses `npm` to install and manage tooling.
+
+1.  Verify that you're running `node` version 0.12 or above and `npm` 
+    version 2.11 or above.
+
+        node -v
+        v0.12.5
+
+        npm -v
+        2.12.2
+
+1.  Install Gulp and Bower.
+
+        npm install -g gulp bower
+
+    Note: the `-g` flag installs Gulp and Bower globally, so you may need to 
+    execute the script with `sudo` privileges. The reason they are installed
+    globally is because some scripts in the PSK expect
+    `gulp` and `bower` to be available from the command line. 
+
+1.  Download the [latest PSK release](https://github.com/PolymerElements/polymer-starter-kit/releases/latest).
+   
+    There are two versions of the PSK, a light version (e.g. 
+    `polymer-starter-kit-light-x.x.x.zip`)
+    and a full version (e.g. `polymer-starter-kit-x.x.x.zip`). Download 
+    the full
+    version.
+
+1.  Unzip the file to a suitable location. After unzipping the file 
+    you should have a directory called `polymer-starter-kit-x.x.x`. 
+    You can rename the directory to something more relevant to your project.
+
+1. `cd` into the base directory of your PSK project.
+
+1.  Install the build and toolchain dependencies.
+
+        npm install
+
+1.  Install the application dependencies.
+
+        bower install
+
+## Directory structure 
+
+The diagram below is a brief summary of the directories within the PSK. The
+appendix of this tutorial contains a complete overview of the entire PSK
```

Comment:
I feel a little weird about the term "Appendix," because we don't use it anywhere else, and in this case it's a section inside the same page. Maybe just, "See the end of this tutorial for a more complete list of files"?

The complete list may prove to be a maintenance headache depending on how much churn we see in PSK, but we can always pull it if it proves painful. (Or we could migrate it into a text file in the PSK project itself that we could link out to--in which case the engineers are more motivated to keep it up to date themselves.)